### PR TITLE
feat(trading): reset console to default

### LINKED ISSUE #5343: Restore to defaults from settings
## Story

As a user
I want to clear all stored state
So that I can easily troubleshoot issues with persisted settings

## Acceptance Criteria

- [x] I can click a button in the settings sidebar
- [x] I can see a warning message that says you will lose all persisted settings and be logged out
- [x] After I've clicked all localstorage should be cleared (table state, wallet state etc)

## Tasks

- [ ] UX (if needed)
- [ ] Design (if needed)
- [ ] Team and stakeholder review
- [ ] Specs reviewed and created or adjusted
- [ ] Implementation
- [ ] Testing (unit and/or e2e)
- [ ] Code review
- [ ] QA review

### PR DESCRIPTION
**Pull request by MadalinaRaicu**
# Related issues 🔗

Closes #5343 

# Description ℹ️

Reset console to default.

# Demo 📺

image

# Technical 👨‍🔧

Clears local storage and hard reloads console.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -52,3 +52,4 @@ cypress.env.json
 /apps/**/cypress/reports/
 /apps/**/cypress/downloads/
 /apps/**/fixtures/wallet/node**
+.nx/
```
Comment (MadalinaRaicu) [inline .gitignore:55]: can we ignore this folder that gets generated in main?

**Review by bwallacee — COMMENTED** [review]
The functionality works well, but this feels like something that we can cover in jest rather than e2e?

**Comment by MadalinaRaicu** [comment]
> The functionality works well, but this feels like something that we can cover in jest rather than e2e?

alright, but e2e tests seem to be failing. is this because this is against `main` and not `develop` ?

**Comment by bwallacee** [comment]
> > The functionality works well, but this feels like something that we can cover in jest rather than e2e?
> 
> alright, but python e2e tests seems to be failing. is this because this is against `main` and not `develop` ?

yeah exactly main, doesnt have the python test work that we have done since being migrated into the monorepo. It's still running against the old console-test that is missing lots of changes.

**Review by bwallacee — APPROVED** [review]
I'm guessing we can't get a test that checks local storage is actually cleared through jest? Once this gets merged back into develop we can get an e2e test to cover it.

**Review by asiaznik — APPROVED** [review]

**Pull request merged**